pom.xml

```
@@ -60,7 +60,7 @@
   </developers>
 
   <properties>
-    <revision>2.4.0-SNAPSHOT</revision>
+    <revision>2.5.0-SNAPSHOT</revision>
```

Version 2.4.0 isn't released yet, so no version bump is needed.
```suggestion
    <revision>2.4.0-SNAPSHOT</revision>
```